src/xbps/repositories/official/index.md

```diff
@@ -3,11 +3,23 @@
 Void provides other official repositories, which are maintained by the Void
 project, but not installed in the default configuration:
```

Comment:
```suggestion
project, but not installed in the default configuration. The following table describes XBPS packages
you can install to enable additional repositories and what those repositories contain.
```
Any time there's a table, there should probably be a line that introduces that table's contents, especially now that the contents of the table are referred to in later writing.